packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -81,7 +81,11 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
 
         this._paymentMethod = state.getPaymentMethodOrThrow(methodId);
 
-        await this._paymentIntegrationService.verifyCheckoutSpamProtection();
+        const checkout = state.getCheckout();
```

Comment:
Can you please check for cart instead of checkout here.

Reply:
fixed